src/Sketch.java

```diff
@@ -19,8 +33,38 @@ public void settings() {
      */
     public void setup() {
         ball1 = new Ball(this);
+        ball1.setColors(color(50, 255, 120), color(100, 195, 165));
+        ball2 = new Ball(this, 15, 300, 250, 1, 2);
+        ball2.setColors(color(150, 50, 80), color(235, 55, 240));
+        ball3 = new Ball(this, 25, 150, 200, -2, -3);
+        ball3.setColors(color(200, 85, 130), color(240, 195, 25));
+        ball4 = new Ball(this, 35, 100, 300, -1, 3);
+        ball4.setColors(color(100, 155, 255), color(240, 125, 255));
+        /** This method gives the balls specific attributes. For the first one, 
```

Comment:
nice comment, helpful and thoughful.

In the future, avoid using the special documentation comment format `/** */` for comments inside of methods like this. Put these comments just before a method, and keep them short and descriptive like `/** This method handles collision when the balls hit */`. Those comments are intended for people *using your code* who may or may not care about how it works internally. If you want to make a longer comment aimed at me or other programmers modifying your code, you can use a regular comment with a single star lik `/* Here are some details about this code... */`